examples/animation/cubic_curve.rs

```diff
@@ -0,0 +1,93 @@
+//! Demonstrates how to work with Cubic curves.
+//!
+
+use bevy::{math::cubic_splines::CubicCurve, prelude::*};
+
+#[derive(Component)]
+pub struct Curve(CubicCurve<Vec3>);
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_systems(Startup, setup)
+        .add_systems(Update, animate_cube)
+        .run();
+}
+
+fn setup(
+    mut commands: Commands,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+) {
+    // Define your control points
+    // These points will define the curve
+    // You can learn more about bezier curves here
+    // https://en.wikipedia.org/wiki/B%C3%A9zier_curve
+    let control_point1 = Vec3::new(-6., 2., 0.);
+    let control_point2 = Vec3::new(12., 8., 0.);
+
+    let control_point3 = Vec3::new(-12., 8., 0.);
+    let control_point4 = Vec3::new(6., 2., 0.);
+
+    let points = [[
+        control_point1,
+        control_point2,
+        control_point3,
+        control_point4,
+    ]];
```

Comment:
This is just a nit, but these lines could be collapsed into a single `points` variable with the comment directly above, a la https://docs.rs/bevy/latest/bevy/math/cubic_splines/struct.Bezier.html#usage/

I don't think the intermediate control point variables add clarity to the example.

```suggestion
    let points = [[
        vec3(-6., 2., 0.),
        vec3(12., 8., 0.),
        vec3(-12., 8., 0.),
        vec3(6., 2., 0.),
    ]];
```